commons-numbers-primes/src/main/java/org/apache/commons/numbers/primes/Primes.java

```diff
@@ -58,20 +58,35 @@ public static boolean isPrime(int n) {
         }
         return SmallPrimes.millerRabinPrimeTest(n);
     }
-
+    /**
+     * Return the number which is not a multiple of 3.
+     *
+     * @param n Positive number.
+     * @return number which is not a multiple of 3.
+     * @throws IllegalArgumentException if n &lt; 0.
+     */
+    public static int nonMultipleOf3(int n) {
+        final int remainder = n % 3;
+        if (0 == remainder) { // if n % 3 == 0
+            n += 2; // n % 3 == 2
+        } else if (1 == remainder) { // if n % 3 == 1
+            n += 4; // n % 3 == 2
+        }
+        return n;
+    }
     /**
      * Return the smallest prime greater than or equal to n.
      *
      * @param n Positive number.
      * @return the smallest prime greater than or equal to {@code n}.
-     * @throws IllegalArgumentException if n &lt; 0.
      */
     public static int nextPrime(int n) {
         if (n < 0) {
             throw new IllegalArgumentException(MessageFormat.format(NUMBER_TOO_SMALL, n, 0));
         }
         if (n <= 2) {
-            return 2;
+            int firstPrime = 2;
```

Comment:
Creating a variable does not add anything to the code. This could all be done using a comment. Note that numbers allows the `magic` numbers beyond the usual set of `{-1, 0, 1}` as the entire library is supporting numerical operations. To create constants for well used constants such as 2 is unnecessary. Note also that loading the numbers -1 to 5 are single [byte code instructions](https://en.wikipedia.org/wiki/List_of_Java_bytecode_instructions). Since direct usage is more efficient, if any of these `int` constants is required then constants should only be used when comments cannot sufficiently clarify their usage.